system-test/read-rows.ts

```diff
@@ -152,6 +149,8 @@ describe('Bigtable/Table', () => {
   });
 
   describe('createReadStream', () => {
+    const INSTANCE = bigtable.instance('instance');
+    const TABLE = INSTANCE.table('table');
```

Comment:
This change is just better factoring. These two variables are only used in this block so should not be mixed with the other test to create confusion.